requirements/requirements.txt

```diff
@@ -1,6 +1,6 @@
 # TTS
 numpy>=1.19.5
-torch>=1.9.0,<3.0.0,!=1.13.0
+torch>=1.9.0,<2.4.0,!=1.13.0
```

Comment:
This is because numpy and torch versions are paired on ARM with some cpp lib
Or just 2.4.0 problematic
No problem, I like `2.3.1` very stable on x86 at least

Reply:
2.3.1 is the last known working version and its [failing in ARM build tests](https://github.com/NeonGeckoCom/NeonCore/actions/runs/10189998167/job/28526725289#step:6:1653) with version 2.4.0